nixnet/_enums.py

```diff
@@ -2029,3 +2029,27 @@ class FrameType(enum.Enum):
     SPECIAL_DELAY = _cconsts.NX_FRAME_TYPE_SPECIAL_DELAY
     SPECIAL_LOG_TRIGGER = _cconsts.NX_FRAME_TYPE_SPECIAL_LOG_TRIGGER
     SPECIAL_START_TRIGGER = _cconsts.NX_FRAME_TYPE_SPECIAL_START_TRIGGER
+
+
+class Terminal(enum.Enum):
+    "Terminals to import/export from the XNET device"
```

Comment:
Use triple quotes for docstrings.

https://www.python.org/dev/peps/pep-0257/#one-line-docstrings